daemon/daemon.go

```diff
@@ -829,7 +829,7 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	// This needs to be done after the node addressing has been configured
 	// as the node address is required as suffix.
 	// well known identities have already been initialized above
-	go cache.InitIdentityAllocator(&d)
+	cache.InitIdentityAllocator(&d)
```

Comment:
nit: Might be nice to comment that we're explicitly ignoring the return channel for now? Otherwise it's not entirely clear why.

Reply:
Done!